spec/features/register_spec.rb

```diff
@@ -0,0 +1,55 @@
+# spec/basic_spec.rb
+
+require 'page'
+
+
+RSpec.feature "Verify basic query search results" do
+
+  before do
+    @page = Page.new
+    @page.visit 'https://google.com'
+  end
+
+  feature "Upcase input" do
+
+    # not sure if curly bracket syntax is acceptable
+    before { @page.enter_keyword 'Monster' }
+
+    feature "First result" do
+      scenario "Title contains inputted keyword Upcase" do
+        expect(@page.first_result_title).to include('Monster')
+      end
+
+      scenario "URL contains inputted keyword Downcase" do
+        expect(@page.first_result_url).to include('monster')
+      end
+
+      scenario "Description contains inputted keyword Upcase" do
+        expect(@page.first_result_description).to include('Monster')
+      end
+    end
+
+  end
+
+
+  feature "Downcase input" do
+
+    before { @page.enter_keyword 'monster' }
+
+    feature "First result" do
+      scenario "Title contains inputted keyword Upcase" do
+        expect(@page.first_result_title).to include('Monster')
```

Comment:
@nonkor , yes. the register in description and title doesn't correlate with the initial query(keyword) register. so as long as we check for hardcoded values this check expects perticular register. the test will become falling once search engine changes the first result. and this is the weak point I don't know how to deal with. maybe we should refuse doing register checks for the title and description sections? but probably there's already a conventional solution for such cases.